docs/content/latest/deploy/checklist.md

```diff
@@ -139,6 +139,18 @@ In practice, the clock drift would have to be orders of magnitude higher in orde
 
 {{< /note >}}
 
+## DNS caching
```

Comment:
Sounds like this applies only to YCQL because it mentions TTL. If only applicable to YCQL, then maybe the title should be "DNS caching [YCQL only]".

Reply:
Yes TTL of DNS cache. Marking as resolved.